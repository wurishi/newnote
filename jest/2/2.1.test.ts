/// <reference path="../node_modules/@types/jest/index.d.ts" />

it('snap correctly', () => {
  const tree = `<div>
  <h1>标题 title</h1>
  </div>`;
  expect(tree).toMatchSnapshot();
});

test('snap test', () => {
  const tree = `<div>
  <h2>Hello World</h2>
  </div>`;
  expect(tree).toMatchSnapshot();
});

// test('snap test2', () => {
//   const tree = '<hr />';
//   expect(tree).toMatchInlineSnapshot();
// });

test('snap Math.random', () => {
  const obj = {
    name: 'Bee',
    id: Math.random(),
  };
  expect(obj).toMatchSnapshot({
    id: expect.any(Number),
  });
});
