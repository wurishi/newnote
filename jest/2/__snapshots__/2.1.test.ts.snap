// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snap Math.random 1`] = `
Object {
  "id": Any<Number>,
  "name": "Bee",
}
`;

exports[`snap correctly 1`] = `
"<div>
  <h1>标题 title</h1>
  </div>"
`;

exports[`snap test 1`] = `
"<div>
  <h2>Hello World</h2>
  </div>"
`;
